### ISSUE TITLE
[Crash][Bug] Fenix crashes when renaming a collection in landscape/portrait mode

### ISSUE THREAD

**Issue by abodea** [mozilla-mobile/fenix#4180]
## Steps to reproduce
1. Launch Fenix.
2. Create any collection with 1 ore more tabs.
3. Set the device in landscape mode.
4. Rename the collection.
5. Tap ok to confirm the new name.
### Expected behavior
The collection name is changed and Fenix works.
### Actual behavior
Fenix crashes after renaming a collection in landscape mode.
### Device information

* Android device: **`Samsung Galaxy S10+(Android 9), LG Nexus 5 (Android 6.0.1), Xiaomi Mi Pad 2 (Android 5.1), HUAWEI MediaPad M3 Lite 10 (Android 7)`**
* Fenix version: **`v1.1.0 rc.1 and Nightly from today 7/19`** but it doesn't occur on the Nightly from **`7/18`**.
### Notes
Note that for some devices after the crash occurred in landscape mode, it started to also occur in the portrait mode too.
[Video](https://drive.google.com/file/d/1ylEjT6FH7_nWqfmv1oLYWduXpNLeo0UH/view?usp=sharing)
[logcat.txt](https://github.com/mozilla-mobile/fenix/files/3411339/logcat.txt)
[logcat2.txt](https://github.com/mozilla-mobile/fenix/files/3411365/logcat2.txt)

**Comment by abodea** [comment]
```css
 D/TimaKeyStoreProvider: TimaKeyStore is not enabled: cannot add TimaSignature Service and generateKeyPair Service
 D/WindowManager: reportFocusChangedSerialized, focused=false, inTouchMode=true, win=Window{3c6ff3a u0 org.mozilla.fenix/mozilla.components.lib.crash.prompt.CrashReporterActivity}
 I/ActivityManager: DSS on for org.mozilla.fenix and scale is 1.0
 D/ViewRootImpl@100817[CrashReporterActivity]: MSG_WINDOW_FOCUS_CHANGED 0 1
 D/ViewRootImpl@1e4dec5[fenix]: MSG_RESIZED_REPORT: frame=Rect(0, 0 - 1440, 3040) ci=Rect(0, 146 - 0, 168) vi=Rect(0, 146 - 0, 168) or=1
 D/InputMethodManager: prepareNavigationBarInfo() DecorView@6d9a51f[CrashReporterActivity]
```
```css
D/ViewRootImpl@100817[CrashReporterActivity]: ViewPostIme pointer 0
 D/InputReader: Input event(10): value=0 when=103511.914452
 D/InputReader: Input event(10): value=0 when=103511.914452
 I/InputReader: Touch event's action is 0x1 (deviceType=0) [pCnt=1, s=] when=103511.914452
 I/clp-JNI: Add pinfo.  (channel) 3c6ff3a org.mozilla.fenix/mozilla.components.lib.crash.prompt.CrashReporterActivity (server) (pkg) AppWindowToken{1ac81a3 token=Token{9526cd2 ActivityRecord{3a35b5d u0 org.mozilla.fenix/mozilla.components.lib.crash.prompt.CrashReporterActivity t249}}} 11592 10237 (action) 1
 I/clp-JNI: Add pinfo end.
 I/InputDispatcher: Delivering touch to (11592): action: 0x1, displayId: 0, '3c6ff3a', toolType: 1
 D/ViewRootImpl@100817[CrashReporterActivity]: ViewPostIme pointer 1
 I/System.out: org.mozilla.fenix
```

**Comment by nikit19** [comment]
This is fixed in #4202

**Comment by boek** [comment]
Thanks for picking this up @nikit19!

**Comment by abodea** [comment]
Verified as fixed in the latest RC build  **`v1.1.0-rc.2`** with 
**`Samsung Galaxy S10+(Android 9) `**
**`Samsung Galaxy S8+(Android 8) `**
**`Sony Xperia Z5(Android 7) `**
**`Samsung Galaxy J7(Android 6) `**
**`Samsung Galaxy Tab A6 (Android 5.1.1)`**.
